assets/MacroAPI.svg

```diff
@@ -0,0 +1,40 @@
2I9Py0+/3pUzWKeQ5pB7VH41GTzuB0RNInQzlEYDPIHanU9ZdaSpvkyvTVFa2Toy8qg0fmU7Xp7ysfiNo6zhB1ddVtv1JJ722EF4VbYZacia5rcxHZxLg2nlVYdnFJr4qSC9WWeg96GQhclu9WPqNoekxORvq6jSmjfTaEKjigirabBKTMu6mgm2MDWHTcymvSu5LEHWQYMh1+cp09U3ymcAkdSKTQnihbZS1FOoas1PJYpFiXXVC5KY6sfTrREjHV6Xr0lTWpHVny7fdUovxnZXVbxST6l6p87qnjfSqHrMLrRejjqtOiNxWz4Ym/2tlO6VYHWUIwZnK6hq7i5GPfb02uamOr06MOo1hGsYQmriHJoivbUEqKlmdJBjz/K5ltro+iNVOlR1OrLE5F51JsPBi1fGR6XRNcluZj06MOo1iWgbKEZNKej7tlLHy2jSOV3U9H5lJ36d8GuDTkTVp5VVBfjHdi02NsSWCA7Q98CQGmuqU6rT2Ylt5dRCeq6wqMlNhFgjMhtgE56pXhytTJ49l+cU4r0luK3Mpd9WZGszin7rJe64jinOaCa+sTFffJPfVmWRUFYDQlDvTVFaHNVan+zHmdxGzXuNIb65PMInEF8PKa4NLs2z9sucxyX11JhlVuq8mKXBFJ28yAjIRXAkkktNj0oivDdGYOnkbydGlzEdn0lH1GE0dRGcrr3Nsr0z7scpd26mqbiuQSM4PkzK+VzZ4os2EF0PXJPfViV2/yU6lKndlFbptcG+GyG3ndyl31fHRi123NZiFwJM6UCf51Ul4VVt+bWkjRCdGHV/U0elUSXQ++rH0Qstjk2SMa3XVCdGdWiRLrnpU+Yx92m6LO9O3TpVWpU95Wf2qUEVHVkd4ed1k56LTBLHF1PFpr442WoFEcs0h9rOPSXh1WXchdeog27K6TSNWB9UmsvPRrZrw6ij3uZcy+lW10RokkmsXYn0fMSIvy5S3lQhtZSF6k4CynVYdZOdbp+6gl7otsibILWY7rUEiuXajrsCHqlyZoWV1kaSvThV1CXV3LnWGk7etTl3jfb56Pm2G6lfdTuuQSG4yMKtkV1WZrd0yulWiyo6o7aRX9bma0qv6vupuq3VIJDdZaAvZVVneBOH56rUVbej4JqFek27REN0Y9epqr3VIJDeZmAWys5U35Z6s6j9TdWfTxLhc2fPWfc1t0y9zbU212Tokkpts1DGOFIMw2l7ue79Nok1v8pNEYDHqN+FWTORWEonkJh91BUzUQXZl26jLLVm2ftuttrJttKVjn1UCb6rtViKR3HRgkoguls60Tu7maNub/DQQWMy2JiW6cWZXP0kkNz1oE9G56tVlHfroxa6rQptdUNPa4U8bubbpHK1GIrnpQZ0uuLqJrom22oI2daqzNr40yZGMM09uhERy04VJJbrYeiHPoan/Qtssg7ZEANbVSU/b+FciN4ZEctOFugMqqrCcmiLPmG1V1dG0YV3CSVobcVYsp0RsBiSSmy7E+D6rCrdvkuhC9ZvANIytTXN0YFsIJRGbIxLJTR/aas21RTfkWqrCpFtm07q0VFsJJBFbABLJTR/qtuZ89dvUdqy6NiQySxZQCBKpRUAiuelD20muav1p+U2nicztOlfVSIRWERLJTR9ifaeTTHQh+k1jEkmtzQE2bUYitBqRSG460YQ1F1KnDuJq82980sgpbe/ijkRkLUEiuenEpJBcSJ227R7gi0kjp2kaj4uFRGAThERy04mmXJZtrxN6fWUxaxbbpJBAIqsZQCK56cSkkVzd54pVX4cmrZ9pG59ry/kSJhSJ5KYTTazy0USd0OuL2VZb1jec9DG6Js+RMMVIJDe9aNKaC6036eNtZTCLY3V1tJkw4+jO+gNIaBWKEtbUJBPdrFtvidwSKkOy5KYXTbvxmqhX5pxNYNatt0RuCZUjkdz0og0k11TdaV+IedIJLpFbQm3opEc91ZhloitbtyqklU0SEmpEnh52Qg1osnNu08LAiaQSYSbUjGTJTTdiWzJNzktrahWXspj03boTKSVMNBLJTT/a4rJsQ/2Y12JCGr9SIxFmQu1IJDf9mCZrLmYbsdpt80r8yYpLmHkkkpt+VEEI00x0bUAiuISESEgkNxtomzUXs51pIrtpXvE/kWZCI0jRlQkhiNVhpW1cvkSbnkUipISpQbLkZgNtGsOahHbqRNuIKRFcwlQhkdzsoI1jc5PQVlWYpQjMRJwJjSGR3Oyg7dZc7LaqaC8G2rz+YyKjhKlDWrtytjAJRFdFe1W16YpJWLm/zVMhEhKCkSy52UKVHf0kWWF1EF6VnXvaDSAhwRHJkps9TBp51PUbLXOeukhi0vZyS+SZ0DiSJTd7qJo0JsUlOklIrsSEhEAkS242kYhucjCpBJcINKEVSJbcbKIOspj0MbWmkdyICQkRkCy52cWkE10d7TeBabCwEokmtAbJkptd1EkQ00CoVSIRT0JCRUiW3Gxj2oiuzvPEwCRHZbbhXAkJViSSS6j7NzCNxOqDaSacRHAJrUNyVyY0QQRNkU8T522q40+EkzDzEMmSSxhjlkhHhRjX0RZSSaSakABIJJdAaPK3kH6H5dEkySSCS2gtUueSgGj695B+j/5ommASwSW0GqlTSeCYJhfitGLWXaMJCc5IHUmCCm36XaTf6BO0jVASwSVMBFIHkqBDG38bae3KdiARXMLEIJFcgglt/32kZb3qRyK4hIlCIrkEGyblN5KW9aoeieASJg6J5BJcMYm/lVlf8SQWErklTCwSySX4YJp+L1XeyzSRQiK4hIlGIrmEEKTfzfQjkVvCVCB1VgllkH4/04dEbglThdRJJcRC+i1NLhKxJUwtUseUUAXS76r9SMSWMBNInVFCHUi/s+aRSC1hJpE6n4QmkH531SORWsLMQ6TOJqFlSL9HfyQyS0gwIHUqCZOEtHZlQkKCFxLJJUwz0oonCQmzDCHE/wdpG54dy5jzEgAAAABJRU5ErkJggg=="/>
```

Comment:
Same for this part as well.

Reply:
Unfortunately, its the same reason for why I can't use vector graphics for this.